.golangci.yml

```diff
@@ -1,188 +1,137 @@
-# SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
-#
-# SPDX-License-Identifier: CC0-1.0
```

Comment:
Thanks for your efforts on this PR, could you please add the license notice back?